src/POMDPToolbox.jl

```diff
@@ -16,6 +16,7 @@ import DataStructures: CircularBuffer, isfull, capacity, push!, append!
 using ProgressMeter
 using StatsBase
 using DataFrames
+using ParticleFilters
```

Comment:
Yes it is not obvious. These are the methods introducing the dependency: 
```julia 
unnormalized_util(p::AlphaVectorPolicy, b::AbstractParticleBelief)
action(p::AlphaVectorPolicy, b::AbstractParticleBelief)
value(p::AlphaVectorPolicy, b::AbstractParticleBelief)
```
If we are ok with ParticleFilters depending on POMDPToolbox, I can move these to ParticleFilters. POMDPToolbox has broader usage so it might make sense to have the dependency this way.

Reply:
Yeah, I agree that we should make ParticleFilters dependent on POMDPToolbox and move these functions to ParticleFilters. I think the AbstractParticleBelief specific functions are pretty important because it can be costly to construct a DiscreteBelief.